content/news/2022-05-06-maplibre-rs-weekly.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "maplibre-rs: Weekly Progress"
+date: "2022-05-06"
+categories: ["maplibre-rs"]
+---
+
+[maplibre-rs](https://github.com/maplibre/maplibre-rs) is an upcoming and cross-platform vector map renderer written in Rust. If you want to get in touch with the developers visit the [#maplibre:matrix.org](https://matrix.to/#/#mapr:matrix.org) chat or our Slack at [osmus.slack.com](https://osmus.slack.com/).
+
+## 📰 News
+
+About one months ago the MapLibre organisation aquired the maplibre-rs project (formerly known as mapr). I was happy about the opertunity and the support from the Geo community. It seems like I was not the first with the idea of a map rendering engine in Rust. In the discussion [#18](https://github.com/maplibre/maplibre-rs/discussions/18) a lot of people showed what they have been working on in the past.
```

Comment:
```suggestion
About one month ago, the MapLibre organisation started to host the maplibre-rs project (formerly known as mapr). I was happy about the opportunity and the support from the Geo community. It seems like I was not the first with the idea of a map rendering engine in Rust. In the discussion [#18](https://github.com/maplibre/maplibre-rs/discussions/18) a lot of people showed what they have been working on in the past.
```

Reply:
Since you write this in first person, add something like "written by @maxammann" or so at the beginning of the blog post.